citizen.ts

```diff
@@ -50,17 +62,38 @@ export const issueFirstClassCitizenship = async (
 
   const simnetAddresses = [...simnet.getAccounts().values()];
 
-  const balancesMap = new Map(
+  const stxBalancesMap = new Map(
     Array.from(simnetAddresses, (address) => {
       const balanceHex = simnet.runSnippet(`(stx-get-balance '${address})`);
       return [address, cvToValue(hexToCV(balanceHex))];
     })
   );
 
+  const sbtcBalancesMap = new Map(
+    Array.from(simnetAddresses, (address) => {
```

Comment:
TIL I learn that you can pass a map function to `Array.from`, but instead, I think it as good to just do `simnetAddress.map(address => { ... })`

Reply:
If that’s isomorphic (produces same output) we should do it in all the other places too (in a separate commit/PR). 👍